cmd/envtool/envtool.go

```diff
@@ -54,6 +54,8 @@ var (
 	errorTemplate = template.Must(template.New("error").Option("missingkey=error").Parse(string(errorTemplateB)))
 )
 
+const versionFile = "build/version/version.txt"
```

Comment:
We usually document all unexported consts too, so let's add some description?

For example:

```suggestion
// versionFile contains version information with leading v.
const versionFile = "build/version/version.txt"
```

Reply:
Done!